src/sources/kafka.rs

```diff
@@ -588,6 +588,9 @@ mod integration_test {
                 acknowledgements,
             ));
             let events = collect_n(rx, 10).await;
+            // Wait a beat to let the finalization task collect the
```

Comment:
Typo I think?
```suggestion
            // Wait a bit to let the finalization task collect the
```

Reply:
Yeah, the "beat" was intentional but probably not helpful. I'll reword it to clarify.